cmd/atlas/apply.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var (
+	applyFlags = struct {
+		dsn  string
+		file string
+	}{}
+	// applyCmd represents the apply command
+	applyCmd = &cobra.Command{
+		Use:   "apply",
+		Short: "Apply atlas schema to data source.",
+		Long:  `Apply atlas schema to data source.`,
```

Comment:
Delete one of them (cobra choose one).
```suggestion
		Short: "Apply atlas schema to data source.",
```

Reply:
👍